### PR TITLE
Home Page Customization

### PR DESCRIPTION
**Pull request by BohdanMylyi**

### CODE REVIEW

File: src/utils/animationHomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+      },
+      "-=.3"
+    );
+
+  // change cursor color when nav is open
+  // tl.to(".cursor", {
+  //   delay: -1,
+  //   css: { className: "+=cursor-active" },
+  // }).to(".cursor2", { delay: -1, css: { className: "+=cursor2-active" } });
+};
```
Comment (OliaZakharcuk) [inline src/utils/animationHomePage.tsx:157]: ???

**Review by OliaZakharcuk — COMMENTED** [review]
Don`t you think this buttons look too small? And why do you need the green line under them?

image

**Comment by OliaZakharcuk** [comment]
The background image is of bad quality. Is it possible to fix it?

**Comment by BohdanMylyi** [comment]
this picture has good quality, I just add some blur on it

**Review by OliaZakharcuk — APPROVED** [review]

**Pull request merged**